data/src/main/java/com/linkedin/data/schema/IndentedPdlBuilder.java

```diff
@@ -124,6 +120,15 @@ boolean writeDoc(String doc) throws IOException
     return false;
   }
 
+  @Override
+  PdlBuilder writeJson(Object value) throws IOException
+  {
+    DefaultPrettyPrinter prettyPrinter = new DefaultPrettyPrinter();
+    prettyPrinter.indentObjectsWith(new DefaultIndenter(" ", DefaultIndenter.SYS_LF + getIndentSpaces()));
```

Comment:
Should this be using two spaces? SPACE * INDENT_WIDTH?

Reply:
fixed